12_7_Accessing_Bitcoind_with_Java.md

```diff
@@ -0,0 +1,131 @@
+# 12.7: Accessing Bitcoind with Java
+
+> **NOTE:** This is a draft in progress, so that I can get some feedback from early reviewers. It is not yet ready for learning.
+
+Interacting with the `bitcoind` directly and using command-line `curl` can get simple if you understand how it works, but there's a project [JavaBitcoindRpcClient](https://github.com/Polve/JavaBitcoindRpcClient) that provides the functionality in a Java-API level, making it even easier to interact with your Bitcoin Server.
```

Comment:
This project does not seem to be very well maintained. It has opened issues from 2015.

Reply:
@kallewoof I understand it might look not maintained, but I tried a couple before and this is by far the best/easiest one to use.
It also has closed issues during this year. Most of the requests are to include new methods in the latest versions of Bitcoin Core.

I interacted with the owner yesterday in a PR, and he is replying.